release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -497,6 +497,18 @@ With this release, you can view network information for ingress flows to {produc
 
 For more information, see  xref:../installing/install_config/configuring-firewall.adoc#network-flow-matrix_configuring-firewall[{product-title} network flow matrix].
 
+[id="ocp-release-notes-networking-ovn-kubernetes-secondary-network-localnet-physical-network-name_{context}"]
+==== OVN-Kubernetes secondary network support for sharing a physical network between localnets
+
+When configuring a secondary network that uses the `localnet` topology, you can now specify a physical network name with the `physicalNetworkName` field. By default, this value is set to the name of the localnet. Multiple localnets can share the same physical network.
+
+For more information, see x!ref:../networking/multiple_networks/secondary_networks/configuring-secondary-network-localnet.adoc#nw-localnet-physical-network-name_configuring-secondary-network-localnet[Configuring a secondary network that uses the localnet topology].
+
+[id="ocp-release-notes-networking-network-attachment-definition_{context}"]
+==== Network attachments support dynamic reconfiguration
+
+Previously, the `NetworkAttachmentDefinition` CR was immutable. With this release, you can edit an existing `NetworkAttachmentDefinition` CR. Support for editing makes it easier to accommodate changes in the underlying network infrastructure, such as adjusting the MTU of a network interface. You still must restart any workloads that use the network attachment definition for the network changes to take effect for those pods.
```

Comment:
It's imperative that we mention the concept of the network being in-sync: let's say you have two NADs that both grant access to the same network (i.e. same network name).

If the configurations on these networks are not in sync, when you restart the workloads, they will *not* get the updated values. Only once *all* NADs pointing to the same network are sync'ed - i.e. share the same configuration - will the updated configuration trickle down to the workloads (after a restart).

While the network is **not** in sync, the behavior is undefined - let's say the control planes restarts while the network is not in sync: we do not have any guarantees over which configuration will be picked up.

@jcaamano FYI ; in case you want to say something further.